spec/features/resident_can_locate_problem_spec.rb

```diff
@@ -100,7 +100,7 @@
     expect(page).to have_content('N1 6NU')
   end
 
-  scenario 'performing a search and not selecting an address, prevents form submission', js: true do
+  scenario 'performing a search and not selecting an address, prevents form submission' do
```

Comment:
I think this test can be removed altogether - the non-JS path is tested by `performing a search and not selecting an address` below.

Reply:
Ah - sorry: I was too blasé and didn't actually read the test. Amended the original commit